package.json

```diff
@@ -119,6 +130,7 @@
     ]
   },
   "resolutions": {
-    "cheerio": "1.0.0-rc.10"
+    "cheerio": "1.0.0-rc.10",
+    "webpack": "^5"
```

Comment:
what's the conflict to override the webpack dependencies?

Reply:
Storybook by default currently ships with a dependency to webpack 4. This led to the two major versions 4 and 5 being installed. For some reason yarn decided to hoist webpack 4 as the default one which obviously led to build errors in Gatsby since it relies on webpack 5. With the opt-in into the webpack 5 builder for storybook and this resolution I was able to deal with this.